src/functions_framework/__init__.py

```diff
@@ -221,6 +236,17 @@ def handle_none(rv):
 
         app.make_response = handle_none
 
+        # Handle log severity backwards compatibility
+        import logging  # isort:skip
+
+        logging.info = _LoggingHandler("INFO", sys.stderr)._log_with_severity
+        logging.warn = _LoggingHandler("ERROR", sys.stderr)._log_with_severity
+        logging.warning = _LoggingHandler("ERROR", sys.stderr)._log_with_severity
+        logging.error = _LoggingHandler("ERROR", sys.stderr)._log_with_severity
+        logging.critical = _LoggingHandler("ERROR", sys.stderr)._log_with_severity
+        sys.stdout = _LoggingHandler("INFO", sys.stderr)
+        sys.stderr = _LoggingHandler("ERROR", sys.stderr)
```

Comment:
I'm concerned about how we're monkeypatching these. We're only providing `.write()` but since these are normally file objects, there's [many other methods](https://docs.python.org/3/library/io.html#io.TextIOBase) that functions (and dependencies of functions) might be using here (like `.flush()`) that will start failing with this change.

We may want to just monkeypatch `.write()` instead.

Reply:
Had issues just monkeypatching write(), it was causing memory issues in GCF. The solution I have now is to have the LoggingHandler extend IOTextWrapper, so it should have the other methods normally available to stdout/stderr.